scripts/lib/cluster.sh

```diff
@@ -1,15 +1,19 @@
 #!/usr/bin/env bash
 
 function down-test-cluster() {
+    if [[ -n "${CIRCLE_JOB:-}" ]]; then
```

Comment:
Why would you want to disable the cluster delete when running in CI?

Reply:
Just making it more verbose. See comment below.